llm_on_ray/inference/serve.py

```diff
@@ -130,6 +139,13 @@ def main(argv=None):
 
     args = parser.parse_args(argv)
 
+    all_models_name = list(all_models.keys())
+    if args.list_model_ids:
+        for model in all_models_name:
+            print(model)
+            print("config_file_path:" + "llm_on_ray/inference/models/" + model + ".yaml")
```

Comment:
Could you print model id and config file path in the same line?  you can use a f-string. 
```suggestion
            print(f"{model}: \tllm_on_ray/inference/models/{model}.yaml")
```
and the model yaml filename is not always equal to the model-id right? You need to fetch the real path of the file?

Reply:
Okay, i try for if it is more appropriate to define a map